products/sapmachine.md

```diff
@@ -0,0 +1,98 @@
+---
+title: SapMachine
+category: lang
+tags: sap java-distribution
+iconSlug: openjdk
+permalink: /sapmachine
+versionCommand: java -version
+releasePolicyLink: https://github.com/SAP/SapMachine/wiki/Security-Updates,-Maintenance-and-Support
+releaseDateColumn: true
+
+auto:
+-   github_releases: "SAP/SapMachine"
+    regex: '^sapmachine-(?P<version>[\d\.]+)$'
+
+# LTS : EOL dates can be found on https://github.com/SAP/SapMachine/wiki/Security-Updates,-Maintenance-and-Support
+# non-LTS : EOL(x) = releaseDate(x+1) (or EOL(x) + 6 months rounded up to the end of the month if unknown)
```

Comment:
Confused  by the second option.

Reply:
I updated the comment to only leave `EOL(x) = releaseDate(x+1)` for non-LTS. Exact date for the next future release can be  found on https://www.java.com/releases/.